### ISSUE TITLE
 "Typing should have a gap of about 2 characters, and it's better to restart or shut down.

### ISSUE THREAD

**Issue by A-caibird** [marktext/marktext#3627]
"Typing should have a gap of about 2 characters, and it's better to restart or shut down.

**Comment by WilczynskiT** [comment]
Dear @MagicXcode  ,

I hope this message finds you well. I wanted to inform you that the MarkText project has been abandoned. However, I have taken the initiative to reactivate the project under a new repository. You can find the reactivated project here: https://github.com/wilcz-com/wMarkText

I kindly request you to kindly transfer this issue to the new repository and continue the discussion and development efforts there. Your continued support and contributions would be greatly appreciated.

Thank you for your understanding and cooperation.

Best regards,
WilczynskiT

**Comment by Jocs** [comment]
Thank you for the report. Unfortunately this issue does not contain enough information for us to investigate or reproduce the problem. Could you please provide:

1. The MarkText version you were using (Help → About).
2. Your operating system and version.
3. Step-by-step reproduction instructions (what you type, in what kind of block, etc.).
4. Whether the issue appears consistently or only after extended use.
5. Any screenshots or screen recordings showing the lag.

Without these details we are unable to act on this report. We are closing it now, but please feel free to open a new issue with the information above and we will be happy to investigate.
